src/gui_common/CustomRichTextLabel.cs

```diff
@@ -229,28 +229,27 @@ private string BuildTemplateForTag(string input, ThriveBbCode bbcode, List<strin
         {
             case ThriveBbCode.Compound:
             {
-                if (!SimulationParameters.Instance.DoesCompoundExist(input))
-                {
-                    GD.Print($"Compound: \"{input}\" doesn't exist, referenced in bbcode");
-                    break;
-                }
-
-                var compound = SimulationParameters.Instance.GetCompound(input);
-
-                var name = compound.Name;
-
                 var pairs = StringUtils.ParseKeyValuePairs(attributes);
 
-                // Parse attributes if there are any
-                if (pairs.TryGetValue("text", out string value))
+                var internalName = string.Empty;
+
+                if (pairs.TryGetValue("type", out string value))
                 {
                     if (!value.StartsAndEndsWith("\""))
                         break;
 
-                    name = value.Substring(1, value.Length - 2);
+                    internalName = value.Substring(1, value.Length - 2);
+                }
+
+                if (!SimulationParameters.Instance.DoesCompoundExist(internalName))
+                {
+                    GD.Print($"Compound: \"{internalName}\" doesn't exist, referenced in bbcode");
```

Comment:
I think you should still use the old algorithm here as a fallback to not break any current translations and to have it as a fallback

Reply:
Added.

Reply:
We need to get the translations updated at some point...
Either need to mass flag them as fuzzy or manually edit each language.